src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServicePermissionTests.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Security;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.PermissionTestUtil;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class UserManagementServicePermissionTest
+    {
+        [Fact]
+        public void ListPendingInvitesAsync()
+        {
+            PolicyCheckBuilder()
+                .ExpectCheck(SecurityPolicies.CanManageUsersOnSystem)
+                .AssertSuccess(
+                    async userService =>
+                    {
+                        await using (var userAndRolesDbContext =
+                            DbUtils.InMemoryUserAndRolesDbContext(Guid.NewGuid().ToString()))
```

Comment:
You can just use `InMemoryUserAndRolesDbContext()` here and also you can use a static import for `DbUtils` to avoid repeating it in every test.